applications/FluidDynamicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -147,6 +147,20 @@ void AddCustomProcessesToPython(pybind11::module& m)
     .def(py::init< ModelPart&, Parameters&, Process::Pointer >())
     .def("SmoothDistanceField", &TwoFluidsInletProcess::SmoothDistanceField)
     ;
+
+    py::class_<DistanceSmoothingProcess<2,SparseSpaceType,LocalSpaceType,LinearSolverType>, DistanceSmoothingProcess<2,SparseSpaceType,LocalSpaceType,LinearSolverType>::Pointer, Process>(m,"DistanceSmoothingProcess2D")
+    .def(py::init< ModelPart&, LinearSolverType::Pointer >())
+    .def(py::init< ModelPart&, Parameters& >())
+    .def(py::init< Model&, Parameters& >())
+    .def("Clear", &DistanceSmoothingProcess<2, SparseSpaceType, LocalSpaceType, LinearSolverType>::Clear)
+    ;
+
+    py::class_<DistanceSmoothingProcess<3,SparseSpaceType,LocalSpaceType,LinearSolverType>, DistanceSmoothingProcess<3,SparseSpaceType,LocalSpaceType,LinearSolverType>::Pointer, Process>(m,"DistanceSmoothingProcess3D")
+    .def(py::init< ModelPart&, LinearSolverType::Pointer >())
+    .def(py::init< ModelPart&, Parameters& >())
+    .def(py::init< Model&, Parameters& >())
```

Comment:
```suggestion
    .def(py::init< ModelPart&, Parameters >())
    .def(py::init< Model&, Parameters >())
```

due to internal memory management `Kratos::Parameters` should be passed by value

Reply:
Thank you for mentioning.